macros/cross_db_shim/spark_shims.sql

```diff
@@ -0,0 +1,3 @@
+{% macro spark__escape_single_quotes(expression) -%}
```

Comment:
This should eventually be added to the spark adapter!

Reply:
i'm honestly floored by the whitespace issue adding spaces